llumnix/entrypoints/bladellm/client.py

```diff
@@ -216,29 +215,27 @@ async def is_ready(self) -> bool:
 
     async def get_request_outputs_loop(self):
         while True:
-            request_outputs_engine = await self.request_output_queue.get()
-            if request_outputs_engine is None:
+            request_responses: List[LlumnixResponseBladeLLM] = await self.request_output_queue.get()
```

Comment:
request_outputs is better, because this function is called get_request_outputs_loop